### ISSUE TITLE
Missing information in Step #7 Database copy from Sandbox request (if applicable)

### ISSUE THREAD

**Issue by manishmoothedath** [MicrosoftDocs/dynamics-365-unified-operations-public#1746]
The service request for copying Sandbox to Production has a lead time of 5 hours for downtime request (excluding the further SLA of 4 hours to action the service request). which is not mentioned anywhere in this article.  e.g. service request raised at 10 am UTC will have downtime starting from 3 pm UTC.
If one tries to schedule the downtime before that, there is an error in LCS - "The downtime start time is required to be at least 5 full hours".

thanks,
Manish

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: b23a0dbf-7870-affb-b07b-53b8ea704371
* Version Independent ID: 02442fdc-5733-1ab3-da37-a1e6dfc89c41
* Content: [Prepare for go-live - Finance & Operations | Dynamics 365 | #MSDyn365FO](https://docs.microsoft.com/en-us/dynamics365/fin-ops-core/fin-ops/imp-lifecycle/prepare-go-live)
* Content Source: [articles/fin-ops-core/fin-ops/imp-lifecycle/prepare-go-live.md](https://github.com/MicrosoftDocs/Dynamics-365-Unified-Operations-Public/blob/live/articles/fin-ops-core/fin-ops/imp-lifecycle/prepare-go-live.md)
* Service: **dynamics-ax-platform**
* Product: ****
* Technology: ****
* GitHub Login: @ClaudiaBetz-Haubold
* Microsoft Alias: **chaubold**

**Comment by josaw1** [comment]
Thank you for your feedback. @ClaudiaBetz-Haubold can you please take a look?

**Comment by josaw1** [comment]
Created bug 384071 to track.

**Comment by ClaudiaBetz-Haubold** [comment]
@Jo Sawyer<mailto:josaw@microsoft.com> – the feedback does not make sense to me. Step 7 in the document clearly calls out the lead time and the downtime:
[cid:image001.jpg@01D5850F.BE2FB960]

Feel free to make a suggestion to reword this to avoid confusion if you deem it necessary.

Thanks,

Claudia Betz-Haubold
Principal Solution Architect | FastTrack for Dynamics 365<https://dynamics.microsoft.com/en-us/fasttrack/>
office:  +49.89.3176.2804 |  mobile:  +49.151.4401.2309
Check this out: Understand the implementation lifecycle<https://docs.microsoft.com/en-us/dynamics365/unified-operations/fin-and-ops/imp-lifecycle/implementation-lifecycle>
[Microsoft]
